### PR TITLE
Add  security scheme type test

### PR DESCRIPTION
**Pull request by sarah2234**
Problem
* The test for OpenApiSecuritySchemeType was missing.

Solve
* Created unit testing regarding
  * None
  * Basic
  * Bearer

Expectations
*  Ensure OpenApiSecuritySchemeType works as expected.

### CODE REVIEW

**Comment by sarah2234** [comment]
@microsoft-github-policy-service agree

**Comment by sarah2234** [comment]
I am not sure how to do testing for None.

File: test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Enums/OpenApiSecuritySchemeTypeTests.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        [DataRow("Bearer")]
+        public void Given_Enum_Should_Have_Member(string memberName)
+        {
+            var members = typeof(OpenApiSecuritySchemeType).GetMembers().Select(p => p.Name);
+
+            members.Should().Contain(memberName);
+        }
+
+        [DataTestMethod]
+        //[DataRow("None", "")]
```
Comment (justinyoo) [inline test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Enums/OpenApiSecuritySchemeTypeTests.cs:25]: Could you take out this commented line?

Comment (justinyoo) [inline test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Enums/OpenApiSecuritySchemeTypeTests.cs:25]: Also we need another test code to test this enum value. Could you add another test method to ensure it?

**Review by justinyoo — CHANGES_REQUESTED** [review]
Thanks for your PR! I left a couple of comments for you to review.

File: test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Enums/OpenApiSecuritySchemeTypeTests.cs (hunk 1 above)
Comment (sarah2234) [reply test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Enums/OpenApiSecuritySchemeTypeTests.cs]: I deleted the commented line.

Comment (sarah2234) [reply test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Enums/OpenApiSecuritySchemeTypeTests.cs]: I added a new test method for None:
```
public void Given_Enum_Should_Have_No_Decorator(string memberName, string displayName)
```

**Review by justinyoo — APPROVED** [review]
LGTM! Thanks for your contribution!

**Pull request merged**